src/Dhall/Diff.hs

```diff
@@ -1088,6 +1104,19 @@ diffExprF l@(TextLit {}) r =
     mismatch l r
 diffExprF l r@(TextLit {}) =
     mismatch l r
+diffExprF (FilePathLit aL) (FilePathLit aR) =
+    diffText (Text.pack aL) (Text.pack aR)
+diffExprF l@(FilePathLit _) r =
+    mismatch l r
+diffExprF l r@(FilePathLit _) =
+    mismatch l r
+-- TODO: Diff maybeHeaders?
```

Comment:
As we discussed offline, we can probably have import resolution fail if the user specifies both `using` and `as Location`